src/components/network/network.scss

```diff
@@ -67,3 +67,7 @@ $gray: #ccc;
     }
   }
 }
+
+#table-btn .btn {
```

Comment:
Do not use global selection. If you know that an element appear within some parent container, use the scss hierarchy to select parent and then child.